File: responses/P535GE-KY8/reverse_digits.rb
# you can write to stdout for debugging purposes, e.g.
# puts "this is a debug message"

def solution(x)
  reverse = 0
  sign = x < 0 ? -1 : 1
  number = x.abs
  while(number > 0) do
    last_digit = number % 10
    reverse = (reverse * 10) + last_digit
    number = number / 10
  end
  reverse * sign
end

File: responses/4BENHU-NUN/reverse_digits.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(X):
    z = ''

    y = str(abs(X))
    for i in y:
        z = i+z

    ans = abs(int(z))

    if X < 0:
        ans = -1*ans

    return (ans)
